dd-sdk-android-internal/api/apiSurface

```diff
@@ -28,6 +28,17 @@ sealed class com.datadog.android.internal.telemetry.InternalTelemetryEvent
     class AddViewLoadingTime : ApiUsage
       constructor(Boolean, Boolean, Boolean, MutableMap<String, Any?> = mutableMapOf())
   object InterceptorInstantiated : InternalTelemetryEvent
+enum com.datadog.android.internal.telemetry.UploadQualityCategories
+  - COUNT
+  - BLOCKER
+  - FAILURE
+enum com.datadog.android.internal.telemetry.UploadQualityBlockers
```

Comment:
should these enums be names in a singular form? Like `UploadQualityCategory` and `UploadQualityBlocker` respectively.